.github/workflows/publish.yml

```diff
@@ -13,15 +13,15 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
 
       - uses: gradle/wrapper-validation-action@v1
 
-      - name: Set up JDK 20
+      - name: Set up JDK 21
         uses: actions/setup-java@v3
         with:
           distribution: temurin
-          java-version: 20
+          java-version: 21
```

Comment:
Is there a specific reason for using 21 here? Shouldn't we use the lowest supported version for publishing packages?